.vib/argo-cd/runtime-parameters.yaml

```diff
@@ -7,7 +7,8 @@ config:
 controller:
   clusterAdminAccess: true
   service:
-    port: 8083
+    ports:
+      metrics: 8083
```

Comment:
Please apply the same change in goss.yaml file: https://github.com/bitnami/charts/blob/7a59ff6fd17ee772b82018f1cd3164651a609fcf/.vib/argo-cd/goss/goss.yaml#L8